pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: test
-version: 0.12.7-dev
```

Comment:
this was something I did in order to install into other projects (was planning on removing prior to my actual pr)